custom_components/rpi_gpio/hub.py

```diff
@@ -82,10 +83,12 @@ def verify_gpiochip(self, path):
 
     def verify_port_ready(self, port: int):
         info = self._chip.get_line_info(port)
-        _LOGGER.debug(f"original port info: {info}")
-        if info.used and info.consumer != DOMAIN:
-            _LOGGER.error(f"Port {port} already in use by {info.consumer}")
-            raise HomeAssistantError(f"Port {port} already in use by {info.consumer}")
+        _LOGGER.debug(f"original port {port} info: {info}")
+        if info.used:
+            if info.consumer != DOMAIN:
+                raise ServiceValidationError(f"Port {port} already in use by {info.consumer}")
```

Comment:
Why raising `ServiceValidationError` we are not exposing any service

Reply:
Fixed. It was a misunderstanding. I didn't understand it should only be used when exposing a service.